Ginger/Ginger/Activities/ActivitiesListViewPage.xaml.cs

```diff
@@ -87,15 +88,18 @@ private void SetListView()
             }
         }
 
-        private void SetListViewData()
+        private async void SetListViewData()
         {
             if (mBusinessFlow != null)
             {
                 //List Data
                 xActivitiesListView.DataSourceList = mBusinessFlow.Activities;
+
                 //List Grouping
                 xActivitiesListView.AddGrouping(nameof(Activity.ActivitiesGroupID));
-                SetSharedRepositoryMark();
+
+                //shared repo indicator
+                await Task.Run(() => this.SetSharedRepositoryMark());                
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)